build.go

```diff
@@ -36,21 +38,33 @@ func (r *Runner) Command(command string, args ...string) error {
 	return nil
 }
 
-func (r *Runner) UpdateYum() error {
-	return r.Command("yum", "update -y")
-}
+func (r *Runner) InstallPackage(packages []string) error {
+
+	if !r.updated {
+		r.Command("yum", "update -y")
```

Comment:
Yes, this is great! So for a given Runner, it will always do `yum update` the first time it tries to install anything, but never again thereafter. That's perfect.

Reply:
thank you for the code 😊!